main.py

```diff
@@ -141,8 +141,10 @@ def _restore_switch(self, switch_id):
 
         if state:
             switch.enable()
+            self.notify_switch_enabled(switch_id)
         else:
             switch.disable()
+            self.notify_switch_disabled(switch_id)
```

Comment:
Why not emit those events directly from `Switch.enable/disable`?

Reply:
We were not able to obtain the controller inside the class Switch to  put the events in the buffer to send it.

Reply:
Got it.